internal/providers/installations.go

```diff
@@ -0,0 +1,113 @@
+//
```

Comment:
I see from the coverage report that there's no coverage here. Can we get some unit tests for this?

Reply:
added some basic coverage, thanks for reminding me!